api/metrics/src/main/java/io/opentelemetry/api/metrics/LongCounterBuilder.java

```diff
@@ -5,14 +5,42 @@
 
 package io.opentelemetry.api.metrics;
 
+import java.util.function.Consumer;
+
 /** Builder class for {@link LongCounter}. */
-public interface LongCounterBuilder extends SynchronousInstrumentBuilder {
-  @Override
-  LongCounterBuilder setDescription(String description);
+public interface LongCounterBuilder {
 
-  @Override
+  /**
+   * Sets the description for this instrument.
+   *
+   * <p>Description stirngs should follw the instrument description rules:
+   * https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument-description
+   */
+  LongCounterBuilder setDescription(String description);
+  /**
+   * Set the unit of measure for this instrument.
```

Comment:
Careful with command form, let's use present tense throughout

Reply:
Understood, should be fixed.